docs/source/changelog.md

```diff
@@ -6,7 +6,9 @@ list and release milestones.
 Release notes for the `space_packet_parser` library
 
 ### v5.0.0 (unreleased)
-- Replace bitstring objects with native Python bytes objects (except for in CSV-based parsing code)
+- Replace bitstring objects with native Python bytes objects
```

Comment:
Leave a note in here that this is the breaking change and anyone using ConstBitStream objects as input to the generator will need to change to the (much prettier) direct input of a binary filelike object.